src/harness/fake_sas.py

```diff
@@ -357,16 +358,36 @@ def do_GET(self):
     self.wfile.write(json.dumps(response))
 
 
-def RunFakeServer(version, is_ecc):
+def RunFakeServer(version, is_ecc, ca_cert,verify_crl):
   FakeSasHandler.SetVersion(version)
   if is_ecc:
     assert ssl.HAS_ECDH
   server = HTTPServer(('localhost', PORT), FakeSasHandler)
-  server.socket = ssl.wrap_socket(
+
+  if ca_cert is not None:
```

Comment:
What purpose does the new code serve? (I'm not a security expert so it's hard for me to understand it quickly.) In general, I'm inclined to keep fake SAS as simple as possible since it is *not* expected to pass the tests.

Reply:
Checks the validity of all certificates in the chain by   attempting to look up valid CRLs.           This change has been removed as the associated test case has been removed.